### DISCUSSION TITLE
Dialogue Options That Teleport You Elsewhere

### DISCUSSION THREAD

**Discussion by Peterodox** [Peterodox/YUI-Dialogue#89] [Data Collection]
Hello everyone! I want to add an optional error prevention mechanism that disables hotkeys and requires mouse clicks when the NPC has some options that will teleport you to another location. And potentially making a new UI to display these options.

To achieve that, I need to know the **GossipOptionID** of these options, which can only be acquired when you encounter these options in person. And I could use your help 😃 

- Open Dialogue UI Settings - UI and enable **Show Copy Text Button**
- When you interact with an NPC or an object that offers teleporting options (e.g. Suramar Portal in the Nighthold), click the Copy Text button and take a screenshot of the new window.
- This window includes all the info I need. Please upload the image here.
- It's okay to post an NPC that has been posted by other people. Some options, despite having the same name and sending you to the same location, may have different IDs depending on your faction.

![WoWScrnShot_112024_235112](https://github.com/user-attachments/assets/d84e88ed-9e32-40b7-a3a0-dbe85e606193)

**Comment by Xpatro** [comment]
https://www.curseforge.com/linkout?remoteUrl=https%253a%252f%252fimgur.com%252fa%252fGDC0V6o

**Comment by Xpatro** [comment]
Legion: Antorus 

(first portal device) = Broken Cliffs
[NPC: 125720] Lightforged Beacon
The Vindicaar awaits your orders, Rybian. Where do we take the fight next?

[OptionID: 47848] The Exhaust
[OptionID: 47849] The Burning Throne
----

(second portal device) = The Exhaust (return)
[NPC: 128303] Lightforged Beacon
The Vindicaar awaits your orders, Rybian. Where do we take the fight next?

[OptionID: 47864] Broken Cliffs
[OptionID: 47865] The Burning Throne
----

(second portal device) = The Exhaust (After both Floor-bosses)
[NPC: 128303] Lightforged Beacon
The Vindicaar awaits your orders, Rybian. Where do we take the fight next?

[OptionID: 47864] Broken Cliffs
[OptionID: 47865] The Burning Throne
------

(Final Portal Device) = The Burning Throne (Return)
[NPC: 128304] Lightforged Beacon
The Vindicaar awaits your orders, Rybian. Where do we take the fight next?

[OptionID: 47867] Broken Cliffs
[OptionID: 47868] The Exhaust
-------

Teleport to final Boss (from NPC: Magni Bronzebeard
[NPC: 128169] Magni Bronzebeard
Are ye ready, Rybian?

[OptionID: 47667] I'm ready.

**Comment by Xpatro** [comment]
Teleports in Raid (The Nighthold)

A. (Entrance) - Arcing Depths
--------------------------------------------------------
[NPC: 116819] Suramar Portal
The portal shimmers with images of the spire.

[OptionID: 47119] Teleport to the Nightspire.
[OptionID: 47121] Teleport to the Font of Night.
[OptionID: 47122] Teleport to the Palace Entrance.
[OptionID: 47123] Teleport to the Palace Courtyard.
--------------------------------------------------------

B. The Palace Entrance - (spawns before 4th Boss) = No Teleports*
- This is just final destination = dont got new orb

C. Palace Courtyard = "KROSUS"
--------------------------------------------------------
[NPC: 116820] Suramar Portal
The portal shimmers with images of the spire.

[OptionID: 47124] Teleport to the Nightspire.
[OptionID: 47126] Teleport to the Font of Night.
[OptionID: 47127] Teleport to the Palace Entrance.
[OptionID: 47128] Teleport to the Arcing Depths.
--------------------------------------------------------

D. (Below "The Nightspire")
----------      ----------      ----------
[NPC: 116662] Suramar Portal
The portal shimmers with images of the spire.

[OptionID: 47108] Teleport to the Font of Night.
[OptionID: 47110] Teleport to the Arcing Depths.
----------      ----------      ----------

E. The Nightspire = "ELISANDE & GUL'DAN"
Note: Orb before (elisande)
--------------------------------------------------------
[NPC: 116667] Suramar Portal
The portal shimmers with images of the spire.

[OptionID: 47111] Teleport to the Palace Entrance.
[OptionID: 47113] Teleport to the Font of Night.
[OptionID: 47114] Teleport to the Arcing Depths.
--------------------------------------------------------

F. (Final Orb) = After Elisande
-----------------------
[NPC: 116667] Suramar Portal
The portal shimmers with images of the spire.

[OptionID: 47111] Teleport to the Palace Entrance.
[OptionID: 47113] Teleport to the Font of Night. ------> "Gul'dan"
[OptionID: 47114] Teleport to the Arcing Depths.
-----------------------

Note: Had to run through dungeon once (some options was added as bosses died)
so wanted FULL List!

**Comment by Peterodox** [comment]
Thanks! I'll see what I can do.

**Comment by Xpatro** [comment]
ULDUAR - Teleport System (Copied)

Yoggsaron
---
The Displacement Pad appears to be active and stable.

[OptionID: 37960] Teleport to the Expedition Base Camp.
[OptionID: 37961] Teleport to the Formation Grounds.
[OptionID: 37962] Teleport to the Colossal Forge.
[OptionID: 37963] Teleport to the Scrapyard.
[OptionID: 37964] Teleport to the Antechamber of Ulduar.
[OptionID: 37965] Teleport to the Shattered Walkway.
[OptionID: 37967] Teleport to the Conservatory of Life.
[OptionID: 37968] Teleport to the Ulduar Tram.
[OptionID: 37969] Teleport to the Spark of Imagination.
[OptionID: 37971] Teleport to the Prison of Yogg-Saron.

Spark of Imagination
---
The Displacement Pad appears to be active and stable.

[OptionID: 37960] Teleport to the Expedition Base Camp.
[OptionID: 37961] Teleport to the Formation Grounds.
[OptionID: 37962] Teleport to the Colossal Forge.
[OptionID: 37963] Teleport to the Scrapyard.
[OptionID: 37964] Teleport to the Antechamber of Ulduar.
[OptionID: 37965] Teleport to the Shattered Walkway.
[OptionID: 37967] Teleport to the Conservatory of Life.
[OptionID: 37968] Teleport to the Ulduar Tram.
[OptionID: 37969] Teleport to the Spark of Imagination.
[OptionID: 37971] Teleport to the Prison of Yogg-Saron.

Ulduar Tram = NO CLICKABLE TP! "end destination"
-----

Convservatory of Life
---
The Displacement Pad appears to be active and stable.

[OptionID: 37960] Teleport to the Expedition Base Camp.
[OptionID: 37961] Teleport to the Formation Grounds.
[OptionID: 37962] Teleport to the Colossal Forge.
[OptionID: 37963] Teleport to the Scrapyard.
[OptionID: 37964] Teleport to the Antechamber of Ulduar.
[OptionID: 37965] Teleport to the Shattered Walkway.
[OptionID: 37967] Teleport to the Conservatory of Life.
[OptionID: 37968] Teleport to the Ulduar Tram.
[OptionID: 37969] Teleport to the Spark of Imagination.
[OptionID: 37971] Teleport to the Prison of Yogg-Saron.

Shattered Walkway
---
The Displacement Pad appears to be active and stable.

[OptionID: 37960] Teleport to the Expedition Base Camp.
[OptionID: 37961] Teleport to the Formation Grounds.
[OptionID: 37962] Teleport to the Colossal Forge.
[OptionID: 37963] Teleport to the Scrapyard.
[OptionID: 37964] Teleport to the Antechamber of Ulduar.
[OptionID: 37965] Teleport to the Shattered Walkway.
[OptionID: 37967] Teleport to the Conservatory of Life.
[OptionID: 37968] Teleport to the Ulduar Tram.
[OptionID: 37969] Teleport to the Spark of Imagination.
[OptionID: 37971] Teleport to the Prison of Yogg-Saron.

Antechamber of Ulduar
---
The Displacement Pad appears to be active and stable.

[OptionID: 37960] Teleport to the Expedition Base Camp.
[OptionID: 37961] Teleport to the Formation Grounds.
[OptionID: 37962] Teleport to the Colossal Forge.
[OptionID: 37963] Teleport to the Scrapyard.
[OptionID: 37964] Teleport to the Antechamber of Ulduar.
[OptionID: 37965] Teleport to the Shattered Walkway.
[OptionID: 37967] Teleport to the Conservatory of Life.
[OptionID: 37968] Teleport to the Ulduar Tram.
[OptionID: 37969] Teleport to the Spark of Imagination.
[OptionID: 37971] Teleport to the Prison of Yogg-Saron.

Scrapyard
---
The Displacement Pad appears to be active and stable.

[OptionID: 37960] Teleport to the Expedition Base Camp.
[OptionID: 37961] Teleport to the Formation Grounds.
[OptionID: 37962] Teleport to the Colossal Forge.
[OptionID: 37963] Teleport to the Scrapyard.
[OptionID: 37964] Teleport to the Antechamber of Ulduar.
[OptionID: 37965] Teleport to the Shattered Walkway.
[OptionID: 37967] Teleport to the Conservatory of Life.
[OptionID: 37968] Teleport to the Ulduar Tram.
[OptionID: 37969] Teleport to the Spark of Imagination.
[OptionID: 37971] Teleport to the Prison of Yogg-Saron.

Colossal Forge
---
The Displacement Pad appears to be active and stable.

[OptionID: 37960] Teleport to the Expedition Base Camp.
[OptionID: 37961] Teleport to the Formation Grounds.
[OptionID: 37962] Teleport to the Colossal Forge.
[OptionID: 37963] Teleport to the Scrapyard.
[OptionID: 37964] Teleport to the Antechamber of Ulduar.
[OptionID: 37965] Teleport to the Shattered Walkway.
[OptionID: 37967] Teleport to the Conservatory of Life.
[OptionID: 37968] Teleport to the Ulduar Tram.
[OptionID: 37969] Teleport to the Spark of Imagination.
[OptionID: 37971] Teleport to the Prison of Yogg-Saron.

Formation Ground
---
The Displacement Pad appears to be active and stable.

[OptionID: 37960] Teleport to the Expedition Base Camp.
[OptionID: 37961] Teleport to the Formation Grounds.
[OptionID: 37962] Teleport to the Colossal Forge.
[OptionID: 37963] Teleport to the Scrapyard.
[OptionID: 37964] Teleport to the Antechamber of Ulduar.
[OptionID: 37965] Teleport to the Shattered Walkway.
[OptionID: 37967] Teleport to the Conservatory of Life.
[OptionID: 37968] Teleport to the Ulduar Tram.
[OptionID: 37969] Teleport to the Spark of Imagination.
[OptionID: 37971] Teleport to the Prison of Yogg-Saron.

Expedition Basecamp
---
The Displacement Pad appears to be active and stable.

[OptionID: 37960] Teleport to the Expedition Base Camp.
[OptionID: 37961] Teleport to the Formation Grounds.
[OptionID: 37962] Teleport to the Colossal Forge.
[OptionID: 37963] Teleport to the Scrapyard.
[OptionID: 37964] Teleport to the Antechamber of Ulduar.
[OptionID: 37965] Teleport to the Shattered Walkway.
[OptionID: 37967] Teleport to the Conservatory of Life.
[OptionID: 37968] Teleport to the Ulduar Tram.
[OptionID: 37969] Teleport to the Spark of Imagination.
[OptionID: 37971] Teleport to the Prison of Yogg-Saron.
